RecoHGCal/TICL/plugins/PFTICLProducer.cc

```diff
@@ -36,8 +36,8 @@ PFTICLProducer::PFTICLProducer(const edm::ParameterSet& conf)
 
 void PFTICLProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
-  desc.add<edm::InputTag>("ticlCandidateSrc", edm::InputTag("ticlCandidateFromTrackstersProducer"));
-  descriptions.add("pfTICLProducer", desc);
+  desc.add<edm::InputTag>("ticlCandidateSrc", edm::InputTag("ticlCandidateFromTracksters"));
+  descriptions.add("pfTICL", desc);
```

Comment:
I think the automatic replacement of *Producer in the product names inlcuded this one by mistake, see #29446